django/db/backends/base/creation.py

```diff
@@ -174,7 +174,7 @@ def _create_test_db(self, verbosity, autoclobber, keepdb=False):
                     try:
                         if verbosity >= 1:
                             print("Destroying old test database '%s'..."
```

Comment:
I think the idea of Shai's comment on the other PR was to change this message to say "Destroying old test database for alias '%' (test_db_name)" similar to the other messages and to use a helper function to generate it so it's less error prone.